redhat-developer/red-hat-developers-documentation-rhdh
# chore: tweak the help message to adopt the latest example branch name (release-1.y); also make output a bit quieter and fix a bug in the plugin naming for wrapped janus idp plugins

### What does this PR do?

chore: tweak the help message to adopt the latest example branch name (release-1.y); also make output a bit quieter and fix a bug in the plugin naming for wrapped janus idp plugins

Signed-off-by: Nick Boldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
N/A (or see commit message above for issue number)

### How to test this PR?

Run the script without options:
```
./modules/dynamic-plugins/rhdh-supported-plugins.sh 
```
see:
```

Generate an updated table of dynamic plugins from content in the following two repos, for the specified branch:
* https://github.com/janus-idp/backstage-plugins
* https://github.com/janus-idp/backstage-showcase

Requires:
* jq 1.6+

Usage:

./modules/dynamic-plugins/rhdh-supported-plugins.sh -b stable-ref-branch

Options:
  -b, --ref-branch    : Reference branch against which plugin versions should be incremented, like release-1.y or main
  --clean             : Force a clean GH checkout (do not reuse files on disk)
  -h, --help          : Show this help

Examples:

  ./modules/dynamic-plugins/rhdh-supported-plugins.sh -b release-1.3 --clean

```

### PR Checklist

As the author of this Pull Request I made sure that:


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
